tests/archiving/test_export.py

```diff
@@ -0,0 +1,80 @@
+import os
+import unittest
+from pyiron_base import Project
+from pyiron_base.archiving.export_archive import export_database
+from pyiron_base.archiving import export_archive
+import pandas as pd
+from pandas._testing import assert_frame_equal
+from filecmp import dircmp
+from pyiron_base import PythonTemplateJob
+
+class ToyJob(PythonTemplateJob):
+    def __init__(self, project, job_name):
+        super(ToyJob, self).__init__(project, job_name)
+        self.input['input_energy'] = 100
+    
+    # This function is executed 
+    def run_static(self):
+        with self.project_hdf5.open("output/generic") as h5out:
+            h5out["energy_tot"] = self.input["input_energy"]
+        self.status.finished = True
+
+
+class TestPacking(unittest.TestCase):
+    
+    @classmethod
+    def setUpClass(cls):
+        cls.arch_path = 'archive_folder'
+        cls.arch_path_comp = cls.arch_path+'_comp'
+        cls.pr = Project('test')
```

Comment:
Be careful with the project path, it is recommended to use a file dependent path https://github.com/pyiron/pyiron_base/blob/master/tests/job/test_script.py#L12

Reply:
`cls.arch_path` and `cls.arch_path_comp` are created during the test, wherever the test is executed (I guess in this case the root of `pyiron_base`). The problem was with the one I stored for comparison in the test directory.